packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -357,31 +358,47 @@ export const PlanDetails = ({
       )}
 
       {showExpandButton && (
-        <div className="text-center" data-testid="footer">
-          {detailsHidden ? (
-            <Localized id="plan-details-show-button">
+        // if we can make the button span the full width for better a11y,
+        // remove code between start and end comments in both places and
+        // remove "footer-accordion-button" from test;
+        // else keep code as is below to restore focus ring and match prod
+
+        // start
+        <div
+          className="w-full flex justify-center content-center"
+          data-testid="footer-accordion-button"
```

Comment:
@ashrivastava-qa Not sure of the best way to do this yet, but heads up that some `data-testid` values are changing here (in case that has any possible impact on functional tests).

Reply:
@pdehaan @ashrivastava-qa, we have reverted the `data-testid` back to its original value for the time being - while this button isn't a footer, we will opt to update this in a more appropriate ticket/PR at a later time. Thanks!